transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/resources/jinjava/typeMappings/replayer.j2

```diff
@@ -0,0 +1,20 @@
+{%- import "common/route.j2" as rscope -%}
+{%- import "typeMappings/preserveAll.j2" as preserve -%}
+{%- include "typeMappings/rewriteDocumentRequest.j2" -%}
+{%- include "typeMappings/rewriteBulkRequest.j2" -%}
+{%- include "typeMappings/rewriteCreateIndexRequest.j2" -%}
+
+
+{%- set source_and_mappings = {
+  'request': request,
+  'index_mappings': index_mappings,
+  'regex_index_mappings': regex_index_mappings}
+-%}
+{{- rscope.route(source_and_mappings, request.method + " " + request.URI, flags, 'preserve.make_keep_json',
+    [
+        ('(?:PUT|POST) /([^/]*)/([^/]*)/(.*)', 'rewrite_doc_request',  'rewrite_add_request_to_strip_types'),
```

Comment:
Am I going crazy, or is `rewrite_add_request_to_strip_types` not defined anywhere else in the PR?  At least, CMD+F isn't finding it...

Reply:
That's the name of the feature.  If a user wanted to suppress it, that's the value that they'd reference in the feature map.